services/network_service_test.go

```diff
@@ -26,11 +26,11 @@ import (
 )
 
 var (
-	middlewareVersion     = "0.0.6"
```

Comment:
Do we want to update the middleware version here?

Reply:
according to the comments in types.go this is the version of rosetta-zen

Reply:
it appears the only place the var is used is in the test and then in an implementation for the network/options endpoint, see network_service.go ln 95

Reply:
Let's keep at 0.0.6 to reduce risk of anything breaking
Their latest is also 0.0.10

Reply:
from the code: 
`var (
	// MiddlewareVersion is the version
	// of rosetta-zen. We set this as a
	// variable instead of a constant because
	// we typically need the pointer of this
	// value.
	MiddlewareVersion = "0.0.8"
)`

it's not referring to the underlying version of the Rosetta original, but rather the version of rosetta-zen